reference/OpenAPI.json

```diff
@@ -23992,6 +23996,16 @@
           }
         }
       },
+      "LoyaltiesRewardAssignmentsListResponseBody": {
+        "title": "Loyalties Reward Assignments List Response Body",
+        "type": "object",
+        "description": "Response body schema for **GET** `/loyalties/{campaignId}/reward-assignments`.",
```

Comment:
No właśnie, i tu jest bez `v1` ;)

Daj mi znać, to ja to potem zbiorczo oblecę w jakieś PR-ce.

Reply:
zmieniłem wszedzie